living/tatar.md

```diff
@@ -1,29 +1,165 @@
 # Web Language: Tatar
 
 Additional names:
-- татарча / tatarça
-- 
-
-News:
-- 
-- 
-
-Culture / History:
-- 
-- 
-
-Government:
-- 
-- 
+- татарча / tatarça / татар теле
+
+**News:** 
```

Comment:
Please preserve the original heading formatting to ensure this can be parsed in our workflow.